py3status/modules/diskdata.py

```diff
@@ -4,22 +4,27 @@
 
 Configuration parameters:
     cache_timeout: refresh interval for this module. (default 10)
-    disk: show stats for disk or partition, i.e. `sda1`. None for all disks.
+    disk: show stats for mountpoint, disk or partition, i.e. `sda1`.
+        None for all disks, filtered by `type` parameter.
         (default None)
     format: display format for this module.
         (default "{disk}: {used_percent}% ({total})")
     format_rate: display format for rates value
         (default "[\?min_length=11 {value:.1f} {unit}]")
     format_space: display format for disk space values
         (default "[\?min_length=5 {value:.1f}]")
-    sector_size: size of the disk's sectors.
-        (default 512)
+    sector_size: size of the disk's sectors (normally taken from sysfs).
+        (default None)
     si_units: use SI units
         (default False)
     thresholds: specify color thresholds to use
         *(default {'free': [(0, 'bad'), (10, 'degraded'), (100, 'good')],
         'total': [(0, 'good'), (1024, 'degraded'), (1024 * 1024, 'bad')],
         'used_percent': [(0, 'good'), (40, 'degraded'), (75, 'bad')]})*
+    types: space-separated list of device types to include in "all disks mode".
+        Supported values: `disks` for PATA/SATA disks, `raid` for raid devices,
+        `cdrom` for cdroms, `mmc` for memory cards.
+        (default 'disks')
```

Comment:
`types` should be a list too instead of space-separated list. I assume `[]` meant unfiltered all.

Reply:
you're right, fixed this